schemas/context/profile.schema.json

```diff
@@ -150,6 +150,13 @@
           "description":
             "Indicates the `profile` record is for use in testing/verification purposes and should not be automatically included in normal operation(s).",
           "default": false
+        },
+        "xdm:segments":{
+          "title" : "Segment Membership",
+          "type" : "array",
+          "items" : {
+            "$ref" : "https://ns.adobe.com/xdm/context/segmentMembership"
```

Comment:
Final Edits: All lower case please 'segmentMembership'->'segmentmembership'. The schema file name is already lower case, so just this ref change.

Reply:
Done